Add functions for running `mix test` and `mix test.watch` in a child app

frahugo opened the pull request.

frahugo committed:
feat: Add function for running `mix test` and `mix test.watch` in a child app

nicholasjhenry commented on civilcode.shrc:
```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
 
+function deamtw() {
+	docker-compose exec application mix test.watch
+}
```
Why does this need to be an alias?

frahugo replied:
It does not. Lazy I guess. Will convert to an alias

nicholasjhenry commented on civilcode.shrc:
```diff
@@ -4,9 +4,19 @@ alias dea="docker-compose exec application"
+function deamtwl() {
+	app="$1"; shift
+	docker-compose exec application mix cmd --app $app mix test.watch $*
+}
```
Trying to work out what the `l` stands for :-)

frahugo replied:
`local`.. maybe `deamtwa` for `child app`?

nicholasjhenry reviewed (commented):
looks good -- just a couple of questions here

nicholasjhenry commented:
@frahugo How do you want to proceed with this?

frahugo committed:
refactor: Add a « exec mix cmd in child app » alias

This is a more generic that we can use in many scenarios:

- deamc magasin mix test.watch
- deamc magasin mix test
- deamc magasin mix ecto.migrate

frahugo committed:
Merge branch 'master' into add-mix-cmd-aliases

frahugo committed:
style: Use snake case and prefix utility functions

As mentioned in the style guide: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide
